src/App.vue

```diff
@@ -198,17 +202,26 @@ export default {
   },
 
   computed: {
-    ...mapGetters([
-      'getAuthApiUrl',
-      'getBusinessUrl',
-      'getCreateUrl',
-      'getEntityName',
-      'getLegalType',
-      'getIdentifier',
-      'getRegHomeUrl',
-      'isRoleStaff',
-      'showFetchingDataSpinner'
-    ]),
+    ...mapState(useConfigurationStore,
+      [
+        'getAuthApiUrl',
+        'getBusinessUrl',
+        'getCreateUrl',
+        'getRegHomeUrl'
+      ]),
+
+    ...mapState(useBusinessStore,
+      [
+        'getEntityName',
+        'getLegalType',
+        'getIdentifier'
+      ]),
+
+    ...mapState(useRootStore,
+      [
+        'isRoleStaff',
+        'showFetchingDataSpinner'
+      ]),
```

Comment:
OK, I really like this as it makes clear where the getters are coming from.

Reply:
Yes. This is nice!

Reply:
You already have split up stores, why not just mapStore for some of these? the strings are fairly sensitive I believe - what happens if you mapState (useRootStore, ['badfunction']), I don't think it's detected by typescript? There's probably a better way of doing it - but don't know how yet

Reply:
If I use mapStore, I'll have to take a pass at each getter/action and call the respective store that has it.

For example, for `isRoleStaff`, I'll have to call it in the code via `rootSore.isRoleStaff`. What do you think?

Reply:
I'd like to encourage us to use the `@Getter` decorator in class API components so that get full typings.

Some components are still using options API so this is the only syntax available to them. Oh well.